files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -1093,7 +885,7 @@ class CinnamonMenuApplet extends Applet.TextIconApplet {
 
         this.settings = new Settings.AppletSettings(this, "menu@cinnamon.org", instance_id);
 
-        this.settings.bind("show-places", "showPlaces", this._refreshBelowApps);
+        this.settings.bind("show-places", "showPlaces", () => this._refreshBelowApps);
```

Comment:
```suggestion
        this.settings.bind("show-places", "showPlaces", () => this._refreshBelowApps());
```

This callback isn't called. Missing parentheses.